lib/components/base-components/NormalComponent.ts

```diff
@@ -363,6 +374,15 @@ export class NormalComponent<
     this.schematic_component_id = schematic_component.schematic_component_id
   }
 
+  _missingFootprintError() {
```

Comment:
we don't need this function (functions that only do a simple thing, should just have the simple thing inline at the original function all site)